src/rest/subhandlers/not_implemented.erl

```diff
@@ -0,0 +1,16 @@
+%%%-------------------------------------------------------------------
+%%% @author pawel
+%%% @copyright (C) 2021, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 25. May 2021 21:27
+%%%-------------------------------------------------------------------
+-module(not_implemented).
```

Comment:
Po co jest ten plik? Zupełnie nie rozumiem, po co nam jest moduł, który dosłownie nie implementuje żadnej funkcji

Reply:
Jest on zupełnie tymczasowy. W gmm_http_server.erl podaje się do ścieżek nazwę modułu który jest handlerem, i tam gdzie jeszcze nic nie napisałem podaję "not_implemented". Mogłem po prostu nie zrobić modułu i wywalało by się na tym, ale stwierdziłem że zrobię sobie taki moduł na razie, żebym to widział, a wywalę go potem